cmd/buildah/addcopy.go

```diff
@@ -38,20 +34,13 @@ func addAndCopyCmd(c *cli.Context, extractLocalArchives bool) error {
 		name = c.String("name")
 	}
 	root := c.String("root")
-	link := ""
-	if c.IsSet("link") {
-		link = c.String("link")
-		if link == "" {
-			return fmt.Errorf("link location can not be empty")
-		}
-	}
 	dest := ""
 	if c.IsSet("dest") {
 		dest = c.String("dest")
 	}
-	if name == "" && root == "" && link == "" {
+	if name == "" && root == "" {
 		if len(args) == 0 {
-			return fmt.Errorf("either a container name or --root or --link, or some combination, must be specified")
+			return fmt.Errorf("either a container name or --root, or some combination, must be specified")
```

Comment:
While they're different attributes (containers have exactly one ID, that's a hex representation of a 256-bit random number, while they can have zero or more names), the storage library handles name-to-ID lookups for us, so for the most part they're interchangeable.